ext/zlib/tests/zlib_get_coding_type.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+string zlib_get_coding_type ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
```

Comment:
We do not even support PHP 5.3, so this SKIPIF should not be there. Active branches are: 5.6, 7.0, 7.1 and master (7.2)